src/components/dialogs/LoadCorrectionDialog.vue

```diff
@@ -46,6 +46,6 @@ export default class LoadCorrectionDialog extends Vue {
   @Prop() private attach: string
 
   // Pass click event to parent.
-  @Emit() private exit () { }
+  @Emit('close') private exit () { }
```

Comment:
I think will break the "exit" handler in Correction.vue (line 9).

Reply:
Hmm, yea that is a little strange. Going to look into this

Reply:
Updated 👍

Reply:
Fixed